### DISCUSSION TITLE
How abou to add someting like clan war

### DISCUSSION THREAD

**Discussion by nicksttar** [codewars/codewars.com#2902] [General]
it would be interesting how it seems to me to add clan wars so that a project is given one for two clans and they write its implementation in a limited time, and the winner will be chosen according to some criteria or leave two decisions from each clan so that codewars members decide whose clan won.

**Comment by iris-is-me** [comment]
nice concept, you beat me to it

**Comment by umarcoder5** [comment]
Yeah, good idea. It'd make the whole coding process even more entertaining when you are competing along with your team
